<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDialog</class>
 <widget class="QDialog" name="AccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>七牛空间名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AccessKey:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SecretKey:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>域名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bucketLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>七牛空间名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accessKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>AccessKey</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secretKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>SecretKey</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="domainLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>空间绑定域名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
